### ISSUE TITLE
Tests fail with PHP 7.1

### ISSUE THREAD

**Issue by JeroenDeDauw** [DataValues/Geo#72]
https://travis-ci.org/DataValues/Geo/jobs/162094713

**Comment by thiemowmde** [comment]
It reports five errors in GlobeCoordinateParserTest::testParseWithValidInputs, with data sets 31 to 35, e.g.:
```
Array ( [value] => Array ( [latitude] => 55.755786 [longitude] => 37.617633 [altitude] => [precision] => 2.7777777777778E-8 [globe] => http://www.wikidata.org/entity/Q2 ) [type] => globecoordinate )
should equal
Array ( [value] => Array ( [latitude] => 55.755786 [longitude] => 37.617633 [altitude] => [precision] => 2.7777777777778E-8 [globe] => http://www.wikidata.org/entity/Q2 ) [type] => globecoordinate )
```
I believe there is some kind of weird IEEE magic going on. I will try to rewrite the test.

**Comment by thiemowmde** [comment]
Fixed in #74.
